etc/c/odbc/sql.d

```diff
@@ -0,0 +1,1414 @@
+    SQL_IDENTIFIER_CASE               = 28,
+    SQL_IDENTIFIER_QUOTE_CHAR         = 29,
+    SQL_MAX_COLUMN_NAME_LEN           = 30,
+    SQL_MAXIMUM_COLUMN_NAME_LENGTH    = SQL_MAX_COLUMN_NAME_LEN,
+    SQL_MAX_CURSOR_NAME_LEN           = 31,
+    SQL_MAXIMUM_CURSOR_NAME_LENGTH    = SQL_MAX_CURSOR_NAME_LEN,
+    SQL_MAX_SCHEMA_NAME_LEN           = 32,
+    SQL_MAXIMUM_SCHEMA_NAME_LENGTH    = SQL_MAX_SCHEMA_NAME_LEN,
+    SQL_MAX_CATALOG_NAME_LEN          = 34,
+    SQL_MAXIMUM_CATALOG_NAME_LENGTH   = SQL_MAX_CATALOG_NAME_LEN,
+    SQL_MAX_TABLE_NAME_LEN            = 35,
+    SQL_SCROLL_CONCURRENCY            = 43,
+    SQL_TXN_CAPABLE                   = 46,
+    SQL_TRANSACTION_CAPABLE           = SQL_TXN_CAPABLE,
+    SQL_USER_NAME                     = 47,
+    SQL_TXN_ISOLATION_OPTION          = 72,
+    SQL_TRANSACTION_ISOLATION_OPTION  = SQL_TXN_ISOLATION_OPTION,
+    SQL_INTEGRITY                     = 73,
+    SQL_GETDATA_EXTENSIONS            = 81,
+    SQL_NULL_COLLATION                = 85,
+    SQL_ALTER_TABLE                   = 86,
+    SQL_ORDER_BY_COLUMNS_IN_SELECT    = 90,
+    SQL_SPECIAL_CHARACTERS            = 94,
+    SQL_MAX_COLUMNS_IN_GROUP_BY       = 97,
+    SQL_MAXIMUM_COLUMNS_IN_GROUP_BY   = SQL_MAX_COLUMNS_IN_GROUP_BY,
+    SQL_MAX_COLUMNS_IN_INDEX          = 98,
+    SQL_MAXIMUM_COLUMNS_IN_INDEX      = SQL_MAX_COLUMNS_IN_INDEX,
+    SQL_MAX_COLUMNS_IN_ORDER_BY       = 99,
+    SQL_MAXIMUM_COLUMNS_IN_ORDER_BY   = SQL_MAX_COLUMNS_IN_ORDER_BY,
+    SQL_MAX_COLUMNS_IN_SELECT         = 100,
+    SQL_MAXIMUM_COLUMNS_IN_SELECT     = SQL_MAX_COLUMNS_IN_SELECT,
+    SQL_MAX_COLUMNS_IN_TABLE          = 101,
+    SQL_MAX_INDEX_SIZE                = 102,
+    SQL_MAXIMUM_INDEX_SIZE            = SQL_MAX_INDEX_SIZE,
+    SQL_MAX_ROW_SIZE                  = 104,
+    SQL_MAXIMUM_ROW_SIZE              = SQL_MAX_ROW_SIZE,
+    SQL_MAX_STATEMENT_LEN             = 105,
+    SQL_MAXIMUM_STATEMENT_LENGTH      = SQL_MAX_STATEMENT_LEN,
+    SQL_MAX_TABLES_IN_SELECT          = 106,
+    SQL_MAXIMUM_TABLES_IN_SELECT      = SQL_MAX_TABLES_IN_SELECT,
+    SQL_MAX_USER_NAME_LEN             = 107,
+    SQL_MAXIMUM_USER_NAME_LENGTH      = SQL_MAX_USER_NAME_LEN,
+    SQL_OJ_CAPABILITIES               = 115,
+    SQL_OUTER_JOIN_CAPABILITIES       = SQL_OJ_CAPABILITIES
+}
+
+enum
+{
+    SQL_XOPEN_CLI_YEAR            = 10000,
+    SQL_CURSOR_SENSITIVITY        = 10001,
+    SQL_DESCRIBE_PARAMETER        = 10002,
+    SQL_CATALOG_NAME              = 10003,
+    SQL_COLLATION_SEQ             = 10004,
+    SQL_MAX_IDENTIFIER_LEN        = 10005,
+    SQL_MAXIMUM_IDENTIFIER_LENGTH = SQL_MAX_IDENTIFIER_LEN
+}
+
+// * SQL_ALTER_TABLE bitmasks *
+enum
+{
+    SQL_AT_ADD_COLUMN     = 0x00000001L,
+    SQL_AT_DROP_COLUMN    = 0x00000002L,
+    SQL_AT_ADD_CONSTRAINT = 0x00000008L
+}
+
+/+ The following bitmasks are ODBC extensions and defined in sqlext.d
+enum : ulong
+{
+    SQL_AT_COLUMN_SINGLE                  = 0x00000020L,

```

Comment:
I actually can not find SQL_AT_COLUMN_SINGLE inside sqlext.d, CTRL-F tells me it only exists right here